packages/components/src/components/SquigglePlayground.tsx

```diff
@@ -477,20 +498,32 @@ export const SquigglePlayground: FC<PlaygroundProps> = ({
 
   return (
     <SquiggleContainer>
-      <Tab.Group>
+      <StyledTab.Group>
         <div className="pb-4">
-          <Tab.List className="flex w-fit p-0.5 mt-2 rounded-md bg-slate-100 hover:bg-slate-200">
-            <StyledTab
-              name={vars.showEditor ? "Code" : "Display"}
-              icon={vars.showEditor ? CodeIcon : EyeIcon}
+          <div className="flex justify-between items-center mt-2">
+            <StyledTab.List>
+              <StyledTab
+                name={vars.showEditor ? "Code" : "Display"}
+                icon={vars.showEditor ? CodeIcon : EyeIcon}
+              />
+              <StyledTab name="Sampling Settings" icon={CogIcon} />
+              <StyledTab name="View Settings" icon={ChartSquareBarIcon} />
+              <StyledTab name="Input Variables" icon={CurrencyDollarIcon} />
+            </StyledTab.List>
+            <PlayControls
+              autoplay={vars.autoplay || false}
+              stale={!vars.autoplay && renderedCode !== code}
```

Comment:
It would be nice if this sort of function were part of the reducer/state object.